docs/rules/prefer_as_const.md

```diff
@@ -0,0 +1,34 @@
+Recommends using const assertion (`as const`) over explicitly specifying literal
+types or using type assertion.
+
+When declaring a new variable of a primitive literal type, there are three ways:
+
+1. adding an explicit type annotation
+2. using normal type assertion (like `as "foo"`, or `<"foo">`)
+3. using const assertion (`as const`)
+
+This lint rule suggests using const assertion, because it will generally lead to
+safer code. For more details about const assertion, see
```

Comment:
```suggestion
This lint rule suggests using const assertion because it will generally lead to
a safer code. For more details about const assertion, see
```
nit